README.md

```diff
@@ -50,7 +50,7 @@ provider:
   webhook:
     image:
       repository: ghcr.io/ionos-cloud/external-dns-ionos-webhook
-      tag: v0.6.3
+      tag: v0.7.0
```

Comment:
after we release this, there will be a new tag ? in this case we need to update this example later in a new PR (and skipping CI)